### ISSUE TITLE
Unable to send a value of 0 for decimal element

### ISSUE THREAD

**Issue by mpetito** [Kentico/content-management-sdk-net#29]
### Brief bug description

The default json.net serialization settings for decimal types *always* includes a decimal point.  This appears to be fine for non-zero numbers.  However, the content management api appears to reject a decimal value of `0.0` which is what is serialized if the decimal value is `0` in the strongly typed model.

### Repro steps

Use the SDK to upsert a content variant with a decimal type element after setting that element to 0.  It will generate the following http request:

```
PUT /projects/<project-id>/items/<item-id>/variants/<language-variant> HTTP/1.1
Host: manage.kenticocloud.com
Content-Type: application/json
Authorization: Bearer <token>

{
	"elements":	{
		"product_sku__price": 0.0,
	}
}
```

The API will respond as follows:

```
{
    "request_id": "42a3324d-8c27-48ff-8005-28b6f1c842b4",
    "error_code": 5,
    "message": "The provided request body is invalid. See the 'validation_errors' attribute for more information and specify a valid JSON object.",
    "validation_errors": [
        {
            "message": "Input string was not in a correct format.",
            "path": "elements.product_sku__price",
            "line": 3,
            "position": 27
        }
    ]
}
```

### Expected behavior

Either the API should accept 0.0 as a valid decimal, or the SDK should submit a valid serialized value for the decimal value of 0.

### Additional context

The JSON.NET serializer should probably be configured to serialize the decimal as a string to avoid any loss of precision, or a custom converter for decimal should serialize 0 as `0`.  Both a json value of `"0.0"` and `0` were tested and work as expected.

https://github.com/Kentico/content-management-sdk-net/blob/ded1797c4be6aadb0f98a5de439528173c8f4e45/KenticoCloud.ContentManagement/Modules/ActionInvoker/ActionInvoker.cs#L24-L27

_Internal reference: DEL-2067_

**Comment by petrsvihlik** [comment]
@alesk-kentico @huluvu21 any thoughts?

**Comment by petrsvihlik** [comment]
@mpetito we're discussing the issue internally. I'll let you know about the result.

**Comment by petrsvihlik** [comment]
Our initial investigation shows that we need to dive deeper. Here's the explanation from one of our devs @alesk-kentico :

> This one is tough. KC backend represents numbers with the Decimal type that does not have a counter-part in some supported languages, e.g. JavaScript. So we can choose to represent numbers as strings in the CM API and make it rather inconvenient to all clients (not mentioning a breaking change). Or we can fix the current implementation to accept different representations of 0 and just accept the fact that from time to time clients can unintentionally change the value or get an out of range exception.

@mpetito we agree with the suggested fix (adjusting the serialization configuration). In the meantime, we'll be investigating how we can fix it in general.

**Comment by petrsvihlik** [comment]
fixed in #31
